.gitignore

```diff
@@ -25,3 +25,6 @@ ibrc-*.tar
 # Temporary files, for example, from tests.
 /tmp/
 out
+result
```

Comment:
why the `result` in .gitignore?

Reply:
I think I just assumed that it would need to be added to `.gitignore`. I didn't think flake would automatically get git to ignore it.

Reply:
We're building the pdf via `nix build`:

https://github.com/Benjamin-Philip/1brc_icfp25/blob/15261d0499c29cd7344ebc7ba0e0be974b8332eb/flake.nix#L68-L97